src/components/organisms/Card/CardImage/__snapshots__/CardImage.test.tsx.snap

```diff
@@ -17,8 +17,7 @@ exports[`<Card.Image /> renders the component with no a11y violations 1`] = `
 
 .c3 {
   height: 100%;
-  width: 100%;
-  background-image: url();
+  width: 100% background-image:url();
```

Comment:
missing a semi-colon

Reply:
Good catch. This is because of SC 5 breaks if the colon is missing. It can be a mess but someday we have to do the jump